tests/test_tester_reification.py

```diff
@@ -0,0 +1,120 @@
+import unittest
+from eclingo.internal_states.internal_control import InternalStateControl
+from eclingo.solver.candidate import Candidate
+from eclingo.solver.tester import CandidateTesterReification
+from clingo.symbol import Function
+from eclingo.solver.generator import GeneratorReification
+import eclingo as _eclingo
+
+# Input Tests
+# echo ":- &k{a}. a." | eclingo --output=reify --reification --semantics c19-1
+# echo ":- k(u(a)). u(a). {k(u(a))} :- u(a)." | clingo --output=reify
+# echo "a. b :- &k{a}." | eclingo  --semantics c19-1 --reification
+# echo "a. b :- &k{a}. {a}." | eclingo  --semantics c19-1 --reification -> Generator test equivalence
+
+""" Helper function to generate candidates for a given program and test them"""
+def tester(program):
```

Comment:
Test for tester_reification works fine, the tester is actually testing the candidates correctly. 

A exception of one minor detail of the asserting for model when is not None, that I'll show next Tuesday.